src/components/map/sectors/MapSectorsList.vue

```diff
@@ -1,38 +1,140 @@
 <template>
-    <template v-if="!isHideAtcType('firs')">
+    <template v-if="!isHideAtcType('firs') && !store.mapSettings.vatglasses?.active">
         <map-sector
             v-for="(sector, index) in firs"
             :key="sector.fir.feature.id as string + index"
             :atc="sector.atc"
             :fir="sector.fir"
         />
     </template>
+
+    <template v-else-if="!isHideAtcType('firs') && store.mapSettings.vatglasses?.active">
+        <template
+            v-for="(countryEntries, countryId) in dataStore.vatglassesActivePositions.value"
+            :key="countryId"
+        >
+            <map-vatglasses-position
+                v-for="(position, positionId) in countryEntries"
+                :key="countryId + '-' + positionId"
+                :position="position"
+            />
+        </template>
+
+
+        <map-overlay
+            class="vatglasses-overlay"
+            :model-value="vatglassesPopupIsShown"
+            :settings="{
+                position: getCoordinates,
+                offset: [15, -15],
+            }"
+            :z-index="20"
+        >
+            <common-popup-block
+                class="aircraft-hover"
+            >
+                <template #title>
+                    Positions
```

Comment:
I mean this is ok, but.

In order for me to do a design for this, I need to somehow find actual controllers from regular VATSIM feed. 

Any change you could add a computed with:
1. Shortened controller (search from both firs and locals)
2. With field containing this: `dataStore.vatglassesActivePositions.value[sector.countryGroupId][sector.vatglassesPositionId]`

After that, I'll be able to make a design myself